src/ngraph/runtime/cpu/cpu_cse.cpp

```diff
@@ -29,7 +29,26 @@ using namespace std;
 
 static bool cse_convertlayout(std::shared_ptr<Node> a, std::shared_ptr<Node> b)
 {
-    return false;
+    NGRAPH_DEBUG << "In cse_convertlayout for " << a->get_name() << " and " << b->get_name();
+
+    auto ar_a = std::static_pointer_cast<runtime::cpu::op::ConvertLayout>(a);
+    auto ar_b = std::static_pointer_cast<runtime::cpu::op::ConvertLayout>(b);
+
+    // gets the tensor layout from the given node
+    auto get_tl = [](std::shared_ptr<Node> node) {
+        auto tensor = node->get_output_tensor_ptr();
+        auto cpu_tl = dynamic_cast<ngraph::runtime::cpu::LayoutDescriptor*>(
```

Comment:
static_cast<> since we know that ConvertLayout can only come from the CPU backend.